src/components/index.d.ts

```diff
@@ -1037,6 +1037,39 @@ export declare const NoraDrawer: {
   }
 }
 
+export declare const Accordion: {
+  ({
+    id,
+    expanded,
+    onToggle,
+    children,
+  }: {
+    id?: string
+    expanded: object
+    onToggle: any
+    children: React.ReactNode
```

Comment:
I think the `Accordion` props need to be updated with the new `toggleChildIsTarget` prop